build/olm_builder.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+CERTIFIED_OPERATOR=${CERTIFIED_OPERATOR:-""}
+VERSION=${VERSION:-""}
+
+SRC_DIR=$(pwd)
+OLM_PATH="${SRC_DIR}"/bundle
+COCKROACH_CHART="${SRC_DIR}"/cockroachdb
+stableCSV="${OLM_PATH}"/manifests/cockroachdb.clusterserviceversion.yaml
+bundleDockerfile="${SRC_DIR}"/build/docker-image/olm-catalog/bundle.Dockerfile
+metaAnnotations="${OLM_PATH}"/metadata/annotations.yaml
+
+
+RELEASE_TAG=v"${VERSION}"
+IMAGE_REGISTRY=${IMAGE_REGISTRY:-"quay.io/cockroachdb"} # TODO: Add project name after registry
+
+COCKROACH_TAG=v$(yq '.appVersion' "${COCKROACH_CHART}"/Chart.yaml)
+QUAY_PROJECT="cockraochdb"
+STABLE_CHANNEL=stable-v"$(cut -d'.' -f1 <<<"${VERSION}")".x
+
+function update_olm_operator() {
+    valuesJSON=$(yq -p yaml -o json "${COCKROACH_CHART}"/values.yaml | jq tostring | sed 's/^.\(.*\).$/\1/')
```

Comment:
Check this should provide proper json of the values.yaml file

Reply:
yes it does. I have tested this both on linux and mac machine